src/lib/orionTypes/EntityTypesResponse.cpp

```diff
@@ -66,11 +66,6 @@ std::string EntityTypesResponse::render(ConnectionInfo* ciP, const std::string&
   std::string out                 = "";
   std::string tag                 = "entityTypesResponse";
 
-  if ((ciP->uriParam["attributesFormat"] == "object") && (ciP->outFormat == JSON))
```

Comment:
Will this change make life difficult for our users?
I guess we need an important note about this in CHANGES_NEXT_RELEASE.